lib/diaspora/message_renderer.rb

```diff
@@ -8,7 +8,7 @@ class Processor
       class << self
         private :new
 
-        def process message, options, &block
+        def process(message, options, &block)
```

Comment:
Okay, admitted. I'm the only one who likes to leave off parens in method definitions.